tests/KrakenClient.PrivateApi.Tests.cs

```diff
@@ -13,9 +13,19 @@ namespace KrakenCore.Tests
 
     public partial class KrakenClientTests
     {
+        private void FailFastOnDummyKey()
+        {
+            if (KrakenClient.DummyApiKey == _client.ApiKey)
+                throw new InvalidOperationException(
+            $@"Please configure {nameof(_client.ApiKey)} and {nameof(_client.PrivateKey)} in {nameof(KrakenFixture)}!
+Use {nameof(KrakenClient)}.{nameof(KrakenClient.DummyApiKey)} and {nameof(KrakenClient)}.{nameof(KrakenClient.DummyPrivateKey)} to test only public API.");
+        }
+
         [Fact]
         public async Task GetAccountBalance()
         {
+            FailFastOnDummyKey();
```

Comment:
I would put it in [SetUp] method to avoid manually adding this method in the beginning of each method.

Reply:
Yes, this should be moved to a ctor since it applies to all the private API tests. Currently both the tests for public and private API share the same class `KrakenClientTests` through `partial` keyword. This doesn't really make sense - they should be separate classes for the setup code to apply correctly.